lib/datadog/tracing/distributed/trace_context.rb

```diff
@@ -309,9 +314,9 @@ def extract_datadog_fields(dd_tracestate)
           [origin, sampling_priority, tags, unknown_fields]
         end
 
-        # Restore `:` back to `=`.
+        # Restore `~` back to `=`.
         def deserialize_tag_value(value)
-          value.tr!(':', '=')
+          value.tr!('~', '=')
```

Comment:
(Just to doublecheck: Is this one ok to mutate, unlike the others that were changed to copy instead?)

Reply:
Yes, `value` always comes from the result of a `String#split` in this case.